tsconfig.json

```diff
@@ -15,5 +15,5 @@
     "allowSyntheticDefaultImports": true
   },
   "include": ["./index.d.ts", "./test/**/*.ts*"],
-  "exclude": ["test/typescript/nonEsModuleInterop/**/*.ts"]
+  "exclude": ["test/typescript/nonEsModuleInterop/**/*.ts", "test/typescript/custom-types"]
```

Comment:
I see that the `custom-types` `tsconfig.json` was added with a `t.test.ts`, but I don't see where it is being tested in one of the scripts e.g.:

```json
    "test:typescript": "tslint --project tsconfig.json",
    "test:typescript:noninterop": "tslint --project tsconfig.nonEsModuleInterop.json",
```

Am I missing something?

Reply:
Yeah, I missed that, thanks!
The IDE will point out the error, but it won't fail on the CI, I'll update the package json